File: TransCoder_tokenized_test_set_functions/java/FIRST_NEGATIVE_INTEGER_EVERY_WINDOW_SIZE_K.java
static void printFirstNegativeInteger ( int arr [ ], int n, int k ) {
  boolean flag;
  for ( int i = 0;
  i < ( n - k + 1 );
  i ++ ) {
    flag = false;
    for ( int j = 0;
    j < k;
    j ++ ) {
      if ( arr [ i + j ] < 0 ) {
        System . out . print ( ( arr [ i + j ] ) + " " );
        flag = true;
        break;
      }
    }
    if ( ! flag ) System . out . print ( "0" + " " );
  }
}

File: TransCoder_tokenized_test_set_functions/python/FIRST_NEGATIVE_INTEGER_EVERY_WINDOW_SIZE_K.py
def printFirstNegativeInteger ( arr, n, k ) :
    for i in range ( 0, ( n - k + 1 ) ) :
        flag = False
        for j in range ( 0, k ) :
            if ( arr [ i + j ] < 0 ) :
                print ( arr [ i + j ], end = " " )
                flag = True
                break
        if ( not ( flag ) ) :
            print ( "0", end = " " )
    
